crates/crates_io_database_dump/src/snapshots/crates_io_database_dump__tests__sql_scripts@export.sql.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: crates/crates_io_database_dump/src/lib.rs
 expression: content
+snapshot_kind: text
```

Comment:
`cargo install cargo-insta` 😉 

the latest release removed `snapshot_kind: text` noise again 😅